connector_jira_servicedesk/models/jira_organization/common.py

```diff
@@ -1,22 +1,19 @@
-# Copyright 2016-2019 Camptocamp SA
+# Copyright 2016-2022 Camptocamp SA
```

Comment:
you can remove the second year or simply use `-Today`

Reply:
Ok, I just updated it